src/oc_erchef/apps/chef_objects/src/chef_s3.erl

```diff
@@ -26,6 +26,8 @@
 -module(chef_s3).
 
 -include("chef_types.hrl").
+% is this needed?
+%-include_lib("erlcloud/include/erlcloud_aws.hrl").
```

Comment:
could we remove this if it is not needed?

Reply:
Done.